README.md

```diff
@@ -60,9 +60,7 @@ aws_elb_request_count_sum{job="aws_elb",load_balancer_name="myotherlb",availabil
 
 All metrics are exported as gauges.
 
-Timestamps from CloudWatch are not passed to Prometheus, pending resolution of
-[#398](https://github.com/prometheus/prometheus/issues/398). CloudWatch has
-been observed to sometimes take minutes for reported values to converge. The
+CloudWatch has been observed to sometimes take minutes for reported values to converge. The
```

Comment:
I'd specifically mention that timestamps will be returned, and that this will cause values to persist for 5m rather than usual staleness semantics.